tests/wake-format/basic/stdout

```diff
@@ -635,9 +635,9 @@ def tarball Unit =
 
     def srcs =
         allSources
-        | filter (! matches `(debian/|wake.spec).*` _.getPathName) # omit packaging files
-        | filter (! matches `(\.circleci/|\.github/).*|\.dockerignore` _.getPathName) # omit CI files
-        | filter (! matches `(.*/)*\.gitignore` _.getPathName) # omit git files
+        | filter (!matches `(debian/|wake.spec).*` _.getPathName) # omit packaging files
+        | filter (!matches `(\.circleci/|\.github/).*|\.dockerignore` _.getPathName) # omit CI files
+        | filter (!matches `(.*/)*\.gitignore` _.getPathName) # omit git files
```

Comment:
Looks like this implementation isn't actually solving the original problem?  Or is this considered a primary expression?

Reply:
This is apparently marked as "outdated" because the *first* commit is being reverted, not because the second is being overridden.